example_data/storm_surge/kyststasjoner_norge.nc2019110700

```diff
@@ -0,0 +1 @@
+/home/jrmet/Desktop/Data/MachineOcean/StormSurge/kyststasjoner_norge.nc2019110700
```

Comment:
I mean, fine if it's accessible on our network, but this isn't ...

Reply:
Additionally: If we intend to use the data for tests at some point, the data should be in a subfolder under a folder named `tests`. For instance `tests/data`.

Reply:
Yes, I know that this is a symlink. This is related to #11 and what we discussed after lunch. I do not really want to put the data on the repo publicly before I have discussed with the owner (who is in holidays). I agree a symlink is annoying, maybe a submodule would be better (see discussion in #11 ). What about discussing this on #11, and then I will apply changes here if / as we think is best?

Reply:
Regarding possible tests in the future: I do not think this should be a problem, right? We should be able to read data from a relative path from the test folder, even if we have to first go down 1 level in the tree? In worst case can always use a symlink there too I suppose. I would think that it is more logical to have the data in their own folder, if it may be used not only by tests but also by example scripts etc.

Reply:
Yes, I saw the issue now. It's helpful to link to related issues in the PR, or in the right hand panel.

I agree with your suggestion in the issue.